packages/components/app/styles/components/side-nav/toggle-button.scss

```diff
@@ -0,0 +1,108 @@
+  border-left-color: transparent;
+  border-top-right-radius: var(--token-side-nav-toggle-button-border-radius);
+  border-bottom-right-radius: var(--token-side-nav-toggle-button-border-radius);
+  transform:
+    translateX(var(--hds-app-sidenav-width-expanded))
+    translateY(24px);
+  cursor: pointer;
+  transition:
+    transform var(--hds-app-sidenav-animation-duration)
+    var(--hds-app-sidenav-animation-easing),
+    padding var(--hds-app-sidenav-animation-duration)
+    var(--hds-app-sidenav-animation-easing);
+
+  &::before {
+    position: absolute;
+    top: calc(var(--token-side-nav-toggle-button-border-radius) * -2);
+    left: calc(var(--token-side-nav-wrapper-border-width) * -1);
+    box-sizing: border-box;
+    width: calc(var(--token-side-nav-toggle-button-border-radius) * 2);
+    height: calc(var(--token-side-nav-toggle-button-border-radius) * 2);
+    border-bottom: var(--token-side-nav-wrapper-border-width) solid var(--token-side-nav-wrapper-border-color);
+    border-left: var(--token-side-nav-wrapper-border-width) solid var(--token-side-nav-wrapper-border-color);
+    border-bottom-left-radius: var(--token-side-nav-toggle-button-border-radius);
+    box-shadow: 0 var(--token-side-nav-toggle-button-border-radius) 0 var(--token-side-nav-color-surface-primary);
+    content: "";
+  }
+
+  &::after {
+    position: absolute;
+    bottom: calc(var(--token-side-nav-toggle-button-border-radius) * -2);
+    left: calc(var(--token-side-nav-wrapper-border-width) * -1);
+    box-sizing: border-box;
+    width: calc(var(--token-side-nav-toggle-button-border-radius) * 2);
+    height: calc(var(--token-side-nav-toggle-button-border-radius) * 2);
+    border-top: var(--token-side-nav-wrapper-border-width) solid var(--token-side-nav-wrapper-border-color);
+    border-left: var(--token-side-nav-wrapper-border-width) solid var(--token-side-nav-wrapper-border-color);
+    border-top-left-radius: var(--token-side-nav-toggle-button-border-radius);
+    box-shadow: 0 calc(var(--token-side-nav-toggle-button-border-radius) * -1) 0 var(--token-side-nav-color-surface-primary);
+    content: "";
+  }
+
+  &:hover,
+  &.mock-hover {
+    padding: calc(var(--token-side-nav-toggle-button-padding-vertical) - var(--token-side-nav-wrapper-border-width)) var(--token-side-nav-toggle-button-padding-horizontal) calc(var(--token-side-nav-toggle-button-padding-vertical) - var(--token-side-nav-wrapper-border-width)) 8px;
+    background-color: var(--token-side-nav-color-surface-interactive-hover);
+
+    &::before {
+      box-shadow: 0 var(--token-side-nav-toggle-button-border-radius) 0 var(--token-side-nav-color-surface-interactive-hover);
+    }
+
+    &::after {
+      box-shadow: 0 calc(var(--token-side-nav-toggle-button-border-radius) * -1) 0 var(--token-side-nav-color-surface-interactive-hover);
+    }
+  }
+
+  &:active,
+  &.mock-active {
+    background-color: var(--token-side-nav-color-surface-interactive-active);
+    border-color: transparent;
```

Comment:
this is what happens on `active` state, probably not what we want:
<img width="535" alt="screenshot_3108" src="https://github.com/hashicorp/design-system/assets/686239/3bc3f699-e8f4-4a7c-9011-57a6b0beca2b">

Reply:
I can't seem to be able to replicate this 🤔 
This is what I see.
<img width="1076" alt="Screenshot 2023-09-27 at 17 14 02" src="https://github.com/hashicorp/design-system/assets/788096/b79f0922-0af0-4834-a999-93077359c6f0">

I tried changing the border color and width but doesn't impact the button. I must be missing something.

Reply:
ah, I see what you mean – I'll review these attributes as there seem to be some leftovers from different refactors.